config/minio.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# start minio server
+minio server /data --console-address ":9001"&
+sleep 5
+# download management utility
+if [[ ! -e "/data/mc" ]]; then
+  curl https://dl.min.io/client/mc/release/linux-amd64/mc --create-dirs -o /data/mc
```

Comment:
should this be using `uname -m` instead of `amd64`? especially for arm64 macos machines?

Reply:
good catch, fixed according to minio's naming convention